cgmanifest.json

```diff
@@ -3523,6 +3533,16 @@
         }
       }
     },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "facter",
+          "version": "4.2.5",
+          "downloadUrl": "https://downloads.puppetlabs.com/facter/facter-4.2.5.gem"
```

Comment:
We still need to update our GitHub check for `SPECS-EXTENDED` but in general all new specs also require an update inside `SPECS/LICENSES-AND-NOTICES/data/licenses.json` and `SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`.

Reply:
When added Extended SPECS to data licenses.json checks are failing. Tests are looking packages to be present under SPECS.

Reply:
Ah, I didn't expect that. In that case let's leave this as it is - I'm hoping to address all license entries soon.